CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## v8.0.3 - 2025-08-12
+
+This is only relevant for Kotlin.
+Fixes page size alignment for all supported linkers.
```

Comment:
Possibly add a newline between these two lines, to isolate the first statement when rendered. Not critical.